randovania/game_description/pickup/pickup_definition/standard_pickup.py

```diff
@@ -53,8 +53,8 @@ class StandardPickupDefinition(BasePickupDefinition):
     hide_from_gui: bool = dataclasses.field(default=False, metadata=EXCLUDE_DEFAULT)
     """Whether this pickup should be hidden in the GUI."""
 
-    must_be_starting: bool = dataclasses.field(default=False, metadata=EXCLUDE_DEFAULT)
-    """Whether the pickup is required be a starting item."""
+    starting_condition: StartingEnum = dataclasses.field(default=StartingEnum.CAN_BE_STARTING, metadata=EXCLUDE_DEFAULT)
+    """How this pickup should be treated by the starting items."""
```

Comment:
*how this pickup should behave in regards to being a starting item
Something like that sounds nicer